docs/dev_install.rst

```diff
@@ -1,20 +1,38 @@
 Installation for development
 ============================
 
-.. code-block:: bash
+#. Clone repo and install requirements:
```

Comment:
Do we want to add documentation on using `venv` module, to isolate the development environment from their default python environment?